publish/publish.proj

```diff
@@ -226,6 +233,48 @@
                                  IgnoredErrorMessagesWithConditional="@(IgnorableErrorMessages)" />
   </Target>
 
+  <Target Name="DoPushCoreHostPackagesToAzure"
+          DependsOnTargets="DoPushStandardPackagesToAzure;DoPushSymbolPackagesToAzure"
+          Condition="'$(OfficialPublish)' == 'true'"/>
+
+  <Target Name="DoPushStandardPackagesToAzure">
+    <PropertyGroup>
+      <RelativePath>$(PackagesRelativePath)</RelativePath>
+      <RelativePath Condition="'$(RelativePath)' == ''">$(ConfigurationGroup)/pkg</RelativePath>
+      <ExpectedFeedUrl>https://$(TransportFeedAccountName).blob.core.windows.net/$(TransportFeedContainerName)/$(RelativePath)/index.json</ExpectedFeedUrl>
+      <PublishFlatContainer Condition="'$(PublishFlatContainer)' == ''">false</PublishFlatContainer>
+    </PropertyGroup>
+    <ItemGroup>
+      <ItemsToPush Remove="*.nupkg" />
+      <ItemsToPush Include="@(_DownloadedStandardPackages)"/>
+    </ItemGroup>
+    <PushToBlobFeed ExpectedFeedUrl="$(ExpectedFeedUrl)"
+                    AccountKey="$(TransportFeedAccessToken)"
+                    ItemsToPush="@(ItemsToPush)"
+                    PublishFlatContainer="$(PublishFlatContainer)"
+                    Overwrite="$(OverwriteOnPublish)" 
+                    IndexDirectory="$(ObjDir)"/>
+  </Target>
+
+  <Target Name="DoPushSymbolPackagesToAzure">
```

Comment:
You may want to consider changing `DoPushSymbolPackagesToAzure` and `DoPushStandardPackagesToAzure` to `GatherSymbolPackages`, `GatherStandardPackages`, and a `PushPackagesToAzure` target that just batches on the `ExpectedFeedUrl` which would be metadata of the `ItemsToPush` itemgroup.

Reply:
Yes can we please be consistent with the target naming, in particular the entry point targets. Lets call it PublishToAzureBlobFeed as in @karajas PR https://github.com/dotnet/corefx/pull/24784/files